pkgs/applications/editors/vim/plugins/update.py

```diff
@@ -84,9 +89,21 @@ def generate_nix(self, plugins: List[Tuple[pluginupdate.PluginDesc, pluginupdate
             f.write("\n}\n")
         print(f"updated {outfile}")
 
+def check_rust_plugins():
+    for plugin in RUST_PLUGINS:
+        nix_attr = f"vimPlugins.{plugin}"
 
+        try:
+            subprocess.run(
+                ["nix-build", "-A", nix_attr],
+                check=True,
+                cwd=Path(__file__).parent / "../../../../../"
+            )
+        except subprocess.CalledProcessError:
+            sys.exit(f"ERROR: Build of {nix_attr} failed!")
 
 def main():
+    check_rust_plugins() 
```

Comment:
Currently this is the first function called, but that's only for fast testing since running the actual plugin update takes a long time. If the general idea here is accepted I'd move it to after `pluginupdate.update_plugins(editor, args)`.